app/assets/javascripts/dashboard/dashboard.js

```diff
@@ -289,7 +289,7 @@ MinionPoller = {
   handleAdminUpdate: function(admin) {
     var $notification = $('.admin-outdated-notification');
 
-    if (State.hasPendingStateNode || State.pendingRemovalMinionId) {
+    if (State.hasPendingStateNode || State.pendingRemovalMinionId || failedToUpdateCount > 0) {
```

Comment:
`failedToUpdateCount` is a function and not a variable. You forgot to call it instead of just referring it. I would also move the comparison to the function and rename the function to just `failedToUpdate`.

Reply:
oops. updated, thanks :smile: